ModelViewer/ModelViewerTest/test.cpp

```diff
@@ -90,7 +90,7 @@ void ModelViewerTest::ResetViewAndShow()
 }
 
 
-void ModelViewerTest::integration()
+void ModelViewerTest::integration()// README: The addition of my settings menu borked this test
```

Comment:
I assume it's just the ordering of menu items.  Add another Key_Down and it should be fine :)